app/controllers/users_controller.rb

```diff
@@ -91,4 +93,12 @@ def require_admin
         redirect_to root_path
       end
     end
```

Comment:
check whether this "end" is extra or not?

Reply:
ok ill check